stunnel/stunnel.mli

```diff
@@ -18,6 +18,8 @@ exception Stunnel_verify_error of string
 
 val crl_path : string
 
+val verify_certificates_ctrl : string
+
```

Comment:
Why is this needed?

Reply:
So xapi knows which file to touch/rm (https://github.com/xapi-project/xen-api/pull/4331/files/6c21163963b4c5564155b17cedf154e108c28c00#diff-01e4b0b8aa12fd6279ff47e010ee9ce0aca146821b90a9f667e6f0e1b89eecf0)

Reply:
A better name might be `verify_cert_flag_path` or similar to indicate that this is a file path.

Reply:
Indeed that would be a better name. I kept the name as it is to reduce churn